src/AVSGateway.js

```diff
@@ -77,7 +77,7 @@ export default class AVSGateway {
 
     if (isOk) {
       try {
-        let textResponsesFromAlexa = parser.extractAlexaTextResponse(payload);
+        let textResponsesFromAlexa = parser.extractAlexaTextResponses(payload);
         if (!textResponsesFromAlexa || !textResponsesFromAlexa.get(0))
```

Comment:
I checked the documentation on parser and it doesn't guarantee a defined list. So I think we should continue checking for an undefined list. 

Checking for isEmpty is not necessary in JS because list.get(0) on an empty list returns falsy.

Added comment explaining why only the first element is checked for emptiness.